python/paddle/fluid/framework.py

```diff
@@ -1629,6 +1629,41 @@ def type(self):
         """
         return self.desc.type()
 
+    def clone(self):
+        """
+        Returns a new static Variable, which is the clone of the original static
+        Variable. It remains in the current graph, that is, the cloned Variable 
+        provides gradient propagation. Calling ``out = tensor.clone()`` is same
+        as ``out = assign(tensor)`` .
+
+        Returns:
+            Variable: The cloned Variable.
+
+        Examples:
+            .. code-block:: python
+
+                import paddle
+
+                paddle.enable_static()
+
+                # create a static Variable
+                x = paddle.static.data(name='x', shape=[3, 2, 1])
+                # create a cloned Variable
+                y = x.clone()
+
+        """
+        main_program = default_main_program()
+        output = main_program.current_block().create_var(
```

Comment:
Better `self.block.create_var(..)` and `apend_op`

Reply:
Done